src/components/Tag.js

```diff
@@ -110,43 +129,72 @@ const Tag = ({ color, disableMenu, slug, text, title, maxValue, ...props }) => {
               '?search[stringTraits][0][name]=' +
               title +
               '&search[stringTraits][0][values][0]=' +
-              text
+              originalValue ?? text
```

Comment:
```suggestion
              originalValue
```

read another comment